<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>462</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_ART"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="actionInstall"/>
    <addaction name="separator"/>
    <addaction name="actionRun"/>
    <addaction name="actionDebug"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionDevices"/>
    <addaction name="actionRestart_adb"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave_All"/>
   <addaction name="separator"/>
   <addaction name="actionBuild"/>
   <addaction name="actionInstall"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionDebug"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionDevices"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="cmdBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_ART">
   <property name="text">
    <string>About ART</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../ART.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="icon">
    <iconset resource="../../ART.qrc">
     <normaloff>:/images/build.png</normaloff>:/images/build.png</iconset>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F9</string>
   </property>
  </action>
  <action name="actionInstall">
   <property name="icon">
    <iconset resource="../../ART.qrc">
     <normaloff>:/images/install.png</normaloff>:/images/install.png</iconset>
   </property>
   <property name="text">
    <string>Install</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F7</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="../../ART.qrc">
     <normaloff>:/images/run.png</normaloff>:/images/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../ART.qrc">
     <normaloff>:/images/debug.png</normaloff>:/images/debug.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
   <property name="shortcut">
    <string>Shift+F9</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../ART.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F2</string>
   </property>
  </action>
  <action name="actionDevices">
   <property name="icon">
    <iconset resource="../../ART.qrc">
     <normaloff>:/images/devices.png</normaloff>:/images/devices.png</iconset>
   </property>
   <property name="text">
    <string>Devices</string>
   </property>
  </action>
  <action name="actionRestart_adb">
   <property name="icon">
    <iconset resource="../../ART.qrc">
     <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Restart adb</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="icon">
    <iconset resource="../../ART.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../ART.qrc"/>
 </resources>
 <connections/>
</ui>
